realm-elasticsearch/lib/realm/elasticsearch/gateway.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'typhoeus'
+require 'elasticsearch'
+require 'realm/health_status'
+require 'realm/persistence/gateway'
+require_relative 'repository'
+
+module Realm
+  module Elasticsearch
+    class Gateway < Realm::Persistence::Gateway
+      def initialize(url:, **options) # rubocop:disable Lint/MissingSuper
```

Comment:
Why do you disable rubocop?

Reply:
There was some issue with super (it's not a new code, just moving around). Anyway it will be resolved in next PR as I removed much functionality from the parent class during refactoring and it won't be needed anymore.